### PR TITLE
Issue 1025

### PR DESCRIPTION
**Pull request by iloveeclipse**
----

Make sure these boxes are checked before submitting your PR -- thank you!

- [x] Added an entry into `CHANGELOG.md` if you have changed SpotBugs code

### CODE REVIEW

File: spotbugs/src/xsl/default.xsl [hunk 1]
```diff
@@ -282,7 +282,7 @@
 		<xsl:copy-of select="$bugTableHeader"/>
 		<xsl:apply-templates select="$warningSet">
 			<xsl:sort select="@abbrev"/>
-			<xsl:sort select="Class/@classname"/>
+			<xsl:sort select="Class[1]/@classname"/>
```
Comment (ThrawnCA) [inline spotbugs/src/xsl/default.xsl]: Does this still work if there is only one element?

Comment (iloveeclipse) [reply spotbugs/src/xsl/default.xsl]: I think yes. At least I see in our build that both single / multiple matches are working now.
I have not much experience with that, but stack overflow suggests this as a viable approach:
https://stackoverflow.com/questions/16928121/how-to-select-first-node-only-in-xslt

Comment (iloveeclipse) [reply spotbugs/src/xsl/default.xsl]: @mkienenb , @paulk-asert : WDYT?

Comment (paulk-asert) [reply spotbugs/src/xsl/default.xsl]: +1

**Pull request merged**